rotkehlchen/api/v1/schemas.py

```diff
@@ -3531,3 +3532,55 @@ class FalsePositveSpamTokenSchema(Schema):
 
 class SpamTokenSchema(Schema):
     token = AssetField(expected_type=EvmToken, required=True)
+
+
+class CalendarCommonEntrySchema(Schema):
+    name = fields.String(required=True)
+    description = fields.String(load_default='')
+    counterparty = fields.String(load_default=None)
+    address = fields.String(load_default=None)
+
+
+class NewCalendarEntrySchema(CalendarCommonEntrySchema):
+    timestamp = TimestampField(required=True)
+
+
+class DeleteCalendarEntrySchema(IntegerIdentifierSchema):
+    ...
+
+
+class UpdateCalendarSchema(NewCalendarEntrySchema, IntegerIdentifierSchema):
+    ...
+
+
+class QueryCalendarSchema(DBPaginationSchema, TimestampRangeSchema, DBOrderBySchema):
+    name = fields.String(load_default=None)
+    description = fields.String(load_default=None)
+    counterparty = fields.String(load_default=None)
+    addresses = fields.List(fields.String, load_default=None)
```

Comment:
This should be typed properly no? Don't we have a BlockchainAddressed field which checks stuff?

Otherwise someone could write invalid addresses as api input here

Reply:
I've been checking and we don't have it. We have in `BlockchainAccountsDeleteSchema` something like it but is an address + blockchain thing

Reply:
So if you do it as a field you can type it properly here too. Please do.